python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -837,16 +837,22 @@ def get_backfills(
         status: Optional[BulkActionStatus] = None,
         cursor: Optional[str] = None,
         limit: Optional[int] = None,
+        created_before: Optional[datetime] = None,
```

Comment:
Originally I was thinking it would be nice to have a Filters object here like `RunFilters`. Reusing `'RunFilters` doesn't really make sense since most of those filters can't be applied to the BulkActions table. We could introduce a `BulkActionsFilters` though

Reply:
My thoughts on `BulkActionsFilters` is that it probably makes sense to do? It would make adding future filters easier. My main hesitation is that it's unclear if there will be future filters